.github/workflows/build.yml

```
@@ -145,7 +152,8 @@ jobs:
     runs-on: ubuntu-latest
 
     container:
-      image: docker.io/library/alpine
+      # riscv64 is currently alpine edge only
+      image: docker.io/library/alpine:${{ matrix.target-arch == 'riscv64' && 'edge' || 'latest'}}
```

Looks like I missed a cosmetic whitespace in my previous comment:

```suggestion
      image: docker.io/library/alpine:${{ matrix.target-arch == 'riscv64' && 'edge' || 'latest' }}
```

---

I didn't notice that either tbh